idea-fare/settings/base.py

```diff
@@ -139,3 +140,11 @@
 TAGGIT_CASE_INSENSITIVE = True
 
 ##############################################################################
+
+################ Django-recaptcha3############################
+RECAPTCHA_PRIVATE_KEY = '6LcYvOMUAAAAALY7HbPDdAUguUmtIbunESCd8RsS'
+RECAPTCHA_PUBLIC_KEY = '6LcYvOMUAAAAAPY-6-3kAELFXZ-ZA6jLH9XAW6St'
```

Comment:
These values should never be public...especially `PRIVATE_KEY`
- also, try to make sure that you regenerate these values since they are on github now